### DISCUSSION TITLE
1.4.5

### DISCUSSION THREAD

**Discussion by yujincheng08** [LSPosed/LSPosed#746] [New Release]
### Changelog
- Fix crash when failed to load a dex
- Fix updating of pm-hidden modules
- Fix updating modules after soft-rebooting
- Fix manager crash on tablet devices

### 更新日志
- 修复加载 dex 失败后的崩溃
- 修复被 pm hide 的模块的更新
- 修复软重启后模块不更新
- 修复平板下管理器崩溃

<hr /><em>This discussion was created from the release <a href='https://github.com/LSPosed/LSPosed/releases/tag/v1.4.5'>1.4.5</a>.</em>

**Comment by Sunnnight** [comment]
![Screenshot_20210612-064919.png](https://user-images.githubusercontent.com/62679334/121755796-af0fd180-cb4a-11eb-8332-bc90b8262349.png)It became like this after the magisk update.
